debian/copyright

```diff
@@ -0,0 +1,530 @@
+Format: https://www.debian.org/doc/packaging-manuals/copyright-format/1.0/
+
+Files: *
+Copyright: 660, York Street, Suite 102, San Francisco, CA 94110 USA
+  2004, 2006, The Linux Foundation and its contributors
+License: Apache-2.0
+
+Files: vendor/github.com/Azure/*
+Copyright: 2015, Microsoft Corporation
+License: Expat
+
+Files: vendor/github.com/Microsoft/go-winio/*
+Copyright: 2015, Microsoft
+License: MIT
```

Comment:
Why do you use `Expat` sometimes and `MIT` somtimes?  They're synonyms.  It looks like [Debian prefers `Expat`][1], so I think we should just use that consistently.

[1]: https://www.debian.org/doc/packaging-manuals/copyright-format/1.0/#license-short-name

Reply:
umm, so I used the `cme` tool to first generate this file and went over it manually as much as I could. Do you know if there's an easier way to handle this? I really don't wanna touch this every time stuff in `vendor/` changes. I love myself way too much to subject myself to this torture :)

Reply:
> umm, so I used the cme tool to first generate this file and went over it manually as much as I could. Do you know if there's an easier way to handle this?

No :/.  My preferred approach to this is to align system packages with dependencies, so somebody else would handle packaging and annotating github.com/Microsoft/go-winio, etc. for you ;).  But the Go ecosystem really likes vendoring, so lots of projects have sloppy versioning and release processes, and the system-packages approach doesn't work so well without those.  Maybe we can patch [`cme`][1] and/or get better metadata landed upstream?  Where is `cme` pulling its data from?

[1]: http://search.cpan.org/dist/App-Cme/bin/cme

Reply:
> by system packages, do you mean distro packages?

Yeah.

> We're not putting this into vanilla Debian, so I'm hoping we can relax this a bit, but IANAL

I expect the required accuracy is up to the risk-tolerance of whoever ends up hosting binaries and the generosity of the copright holders.  In this case I don't have a handle on either, so I'd rather not bend rules.  But if the maintainers feel that binary-hosters won't mind, they could allow some curner-cutting.

Reply:
well in fedora packages so far, we've only been mentioning the license of the main package. Also, if it makes things easier I could look at only providing binaries and disabling -dev packages. @rhatdan @runcom @mrunalp comments?

Reply:
*but because it doesn't distribute CRI-O or its dependencies, it can't violate their licenses.*

Not sure I understand what you mean here. The fedora package does distribute the cri-o binary which is also what the ubuntu package does. So, what applies to fedora should also apply to ubuntu, no? IOW, if I'm doing it wrong in fedora, that needs to be fixed as well.

Reply:
> The fedora package does distribute the cri-o binary...

Ah, in that case you'd need to get the licensing right there too unless whoever is hosting the Fedora binaries is comfortable cutting corners.  Maybe they have a "we don't audit this stuff; file takedown notices or patches if we missed something" policy.